aws/lambda/pytorch-auto-revert/pytorch_auto_revert/__main__.py

```diff
@@ -23,7 +23,22 @@ def setup_logging(log_level: str) -> None:
     numeric_level = getattr(logging, log_level.upper(), None)
     if not isinstance(numeric_level, int):
         raise ValueError(f"Invalid log level: {log_level}")
-    logging.basicConfig(level=numeric_level)
+
+    root_logger = logging.getLogger()
+    root_logger.setLevel(numeric_level)
+
+    if not root_logger.handlers:
+        handler = logging.StreamHandler()
+        handler.setLevel(numeric_level)
+        formatter = logging.Formatter(
+            "%(asctime)s %(levelname)s [%(name)s] %(message)s"
```

Comment:
don't we want to keep the same log format that we've been using all along? 

when running the CI locally it does not match. Do we have the same format in the aws lambda? Maybe we should consider to unify both?